tectonic/dpx-pdfobj.c

```diff
@@ -2028,7 +2028,7 @@ pdf_add_stream_flate (pdf_obj *dst, const void *data, int len)
 
     z.zalloc = Z_NULL; z.zfree = Z_NULL; z.opaque = Z_NULL;
 
-    z.next_in  = (z_const Bytef *) data; z.avail_in  = len;
+    z.next_in  = data; z.avail_in  = len;
```

Comment:
Yes I think it will fix it. I looked at `zconf.h` in zlib 1.2.5 and it is missing `zconst`, so since we do not use it anymore, it should work now.